app/src/modules/settings/routes/data-model/collections/collections.vue

```diff
@@ -197,12 +197,14 @@ export default defineComponent({
 			});
 
 			try {
-				await Promise.all(
-					updatesWithSortValue.map((collection) =>
-						api.patch(`/collections/${collection.collection}`, {
+				api.patch(
+					`/collections`,
+					updatesWithSortValue.map((collection) => {
+						return {
+							collection: collection.collection,
 							meta: { sort: collection.meta.sort, group: collection.meta.group },
-						})
-					)
+						};
+					})
```

Comment:
This may be a little premature optimization but theoretically this approach could end up at "max_payload_size". Perhaps we should consider sending this in batches of a set size.